src/compiler/python_generator.cc

```diff
@@ -66,6 +66,9 @@ using std::vector;
 
 namespace grpc_python_generator {
 
+GeneratorConfiguration::GeneratorConfiguration()
```

Comment:
Do we have a different way of specifying namespace renames for internal code? (or is that what you meant?)